media-libs/amdgpu-pro-vulkan/files/icd_amd64.patch

```diff
@@ -0,0 +1,18 @@
+--- amd_icd64.json	2021-07-28 03:02:33.000000000 +0200
```

Comment:
Just do a/ and b/ so you don't have to use -p0?

Reply:
Unfortunately I don't know enough on how to write patches to understand what you mean x)

Reply:
Done!